src/arrayMethodIndexOf.js

```diff
@@ -6,7 +6,17 @@
  */
 function applyCustomIndexOf() {
   [].__proto__.indexOf2 = function(searchElement, fromIndex) {
```

Comment:
you can set default value for fromIndex here `fromIndex = 0`. Then you won't need to check if fromIndex was passed